charts/copia/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: copia
 description: Copia Helm chart for Kubernetes
 type: application
-version: 0.10.0
-appVersion: v0.10.0
```

Comment:
Was I supported to increment the patch version here? I see in previous PRs we've only update `version` with the patch version

Reply:
We update only the version when there is only a change to the Helm chart itself. If it's a change to the app, we change both.